Tests/CommandsTests/BuildCommandTests.swift

```diff
@@ -12,7 +12,11 @@
 
 import Basics
 @testable import Commands
-@testable import CoreCommands
+
+@_spi(SwiftPMInternal)
+@testable 
```

Comment:
Would be nice to remove all our @testable too IMO (to avoid the rebuild) and use SPI instead.